tests/compute/test_csrmm.py

```diff
@@ -5,6 +5,10 @@
 from test_utils import parametrize_idtype
 import backend as F
 
+if F.backend_name == 'pytorch':
+    import torch
+    torch.backends.cuda.matmul.allow_tf32 = False
```

Comment:
You may want to use the newly added:
```
torch.set_float32_matmul_precision(“highest”)
```
setting in `1.12.0`.
See: https://github.com/pytorch/pytorch/pull/76509 and https://pytorch.org/blog/pytorch-1.12-released/#changes-to-float32-matrix-multiplication-precision-on-ampere-and-later-cuda-hardware

Reply:
Yeah. But DGL supports PyTorch 1.9+, so I use `torch.backends.cuda.matmul.allow_tf32` for a better compatibility. I think they are equivalent as per https://pytorch.org/docs/stable/generated/torch.set_float32_matmul_precision.html.
Another concern is that, in Nvidia's PyTorch container 22.05, the PyTorch version is recognized as 1.12.0, but `torch.set_float32_matmul_precision` is not available...

Reply:
OK, makes sense due to BC-breaking behavior. `22.05` uses [1.12.0a0+8a1a93a](https://github.com/pytorch/pytorch/commit/8a1a93a), which is not the stable `1.12.0` release, but a previous commit (note the alpha in the commit),